edx_lint/files/pylintrc

```diff
@@ -36,28 +249,88 @@ load-plugins=edx_lint.pylint,pylint_django,pylint_celery
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
+    # These should be left to the discretion of the reviewer
+    bad-continuation,
+    invalid-name,
+    misplaced-comparison-constant,
+    file-ignored,
+    too-many-nested-blocks,
+    too-many-statements,
```

Comment:
There was some discussion of this up above (@benpatterson had similar concerns, and @ormsbee and I responded)